### ISSUE TITLE
Issue with mvn gcloud:run

### ISSUE THREAD

**Issue by jamesfmackenzie** [ryankennedy/dropwizard-googlecloudplatform#1]
Firstly, thanks for building this!

When trying to run this locally via mvn gcloud:run, I see the following error:

[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 13.929 s
[INFO] Finished at: 2016-07-03T19:34:22+01:00
[INFO] Final Memory: 18M/462M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.google.appengine:gcloud-maven-plugin:2.0.9.111.v20160527:run (default-cli) on project helloworldservice: The application directory does not exist : D:\GitHub\dropwizard-googlecloudplatform-master\target/helloworldservice-1.0-SNAPSHOT -> [Help 1]

I also get the same with my own app-engine / dropwizard creation - and had tried out your solution in the hope you'd already found a fix for this.

To me it looks like a path bug in gcloud-maven-plugin (GCloudAppRun.java) - so unsure how to fix it.

What are your thoughts?

**Comment by ryankennedy** [comment]
It's complaining about the built artifact, which I don't think I'm referencing anywhere. The one path-specific thing I can see that needs adjusting is the [gcloud_directory](https://github.com/ryankennedy/dropwizard-googlecloudplatform/blob/master/pom.xml#L68) configuration in my pom.xml, but I'd expect that to fail differently (can't find path to `gcloud` or something similar).

Have you already adjusted that for your local environment?

**Comment by jamesfmackenzie** [comment]
Yeah, my gcloud path is good. This issue does not happen with Google's Java 8 Runtime sample
